tnnt/templates/leaderboards.html

```diff
@@ -18,46 +18,19 @@
   <div id="leaderboard-combobox">
     <label for="boards">Select leaderboard: </label>
     <select name="boards" id="boards-combobox">
```

Comment:
UX change request: I think it would be nicer to show all of them as buttons or links rather than hide them in a dropdwon menu.